invoicing_app/tests/tests.py

```diff
@@ -393,14 +393,19 @@ def test_get_query_results(self):
         )
         returns_one_element = 1
         self.assertEqual(
-            str(type(self.my_command_new.get_query_results(query_options_test))),
+            str(type(self.my_command_new.get_query_results(
+                query_options_test, self.my_command_new.query
```

Comment:
nit: While some services/libraries in core do send arguments like this, most of the time we prefer this:
```python
str(type(self.my_command_new.get_query_results(
    query_options_test,
    self.my_command_new.query,  # notice the trailing comma
)))
```

Even so, you should probably use a different assert instead of converting this to a string:
```python
self.assertIsInstance(result, list)
```
https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIsInstance

Reply:
Done!